metabase/metabase
# Improve homepage loading speed

Epic https://github.com/metabase/metabase/issues/21322

Currently there is waterfall-like loading of data required for the new homepage. This PR fixes this issue by using `loadingAndErrorWrapper: false` and manually rendering `<LoadingAndErrorWrapper />` when needed.

How to test:
- Open the homepage, navigate to a different page and come back
- We have e2e and unit tests for all possible homepage states, so it should be safe if CI is green
